src/Seq.Input.RabbitMQ/RabbitMQInput.cs

```diff
@@ -80,6 +80,12 @@ public class RabbitMQInput : SeqApp, IPublishJson, IDisposable
             HelpText = "Whether or not messages should be auto-acknowledged. The default is true.")]
         public bool IsReceiveAutoAck { get; set; } = true;
 
+        [SeqAppSetting(
+            DisplayName = "Dead Letter Exchange",
+            IsOptional = true,
+            HelpText = "Name of dead letter exchange associated with this queue.")]
```

Comment:
```suggestion
            HelpText = "The name of the dead letter exchange associated with this queue. If specified, the exchange will be used when declaring the queue, otherwise no dead lettering will be configured.")]
```